src/components/App.jsx

```diff
@@ -1,48 +1,61 @@
-import React, { useState, createContext } from "react";
-import { Header } from "./Header";
-import ListMovies from "./ListMovies";
-import { Footer } from "./Footer";
-import { MovieDetail } from "./MovieDetail";
+import React from "react";
 import "./global.scss";
-import SnackbarProvider from "react-simple-snackbar";
+import { Home } from "./Home";
+import {
+  Routes,
+  Route,
+  Link,
+  useNavigate,
+  useSearchParams,
+  Navigate,
+} from "react-router-dom";
 
 export const App = () => {
-  const userDetailContext = createContext(null);
-
-  const [userDetails] = useState({
-    name: "Andres",
-    id: "000033",
-  });
-
-  const app = { title: "netflix", subtitle: "roulette" };
-
-  const [selectedMovie, setMovieSelected] = useState(null);
-
-  const movieSelectHandler = (event) => {
-    window.scrollTo(0, 0);
-    setMovieSelected(event);
-  };
-
-  const searchSelected = () => {
-    setMovieSelected(null);
-  };
+  let navigate = useNavigate();
+  let [searchParams, setSearchParams] = useSearchParams();
+  let searchQuery = searchParams.get("searchQuery");
```

Comment:
Can you make an object with query params properties and use spread attribute in Home component occurances bellow?
const queryParams = {
searchQuery: searchParams.get("searchQuery"),
genreQuery: searchParams.get("genre"),
sortByQuery: searchParams.get("sortBy"),
movieQuery: searchParams.get("movie"),
};

<Home {...queryParams} />